packages/components/src/components/contract-card/contract-card-items/turbos-card-body.jsx

```diff
@@ -37,7 +37,11 @@ const TurbosCardBody = ({
 
     return (
         <React.Fragment>
-            <div className={is_mobile ? 'dc-contract-card-items-wrapper--mobile' : 'dc-contract-card-items-wrapper'}>
+            <div
+                className={classNames('dc-contract-card-items-wrapper', {
+                    'dc-contract-card--turbos-recent-positions': is_turbos,
```

Comment:
```suggestion
                    'dc-contract-card--turbos': is_turbos,
```
@hamza-deriv we have to use a general classname here, just --turbos. This card will be used not only in recent positions tab but also on contract details page. And for open-positions we can use an extra suffix because styles there are different.

Reply:
Done